Doc/tutorial/classes.rst

```diff
@@ -114,8 +114,8 @@ accessible.  "Directly accessible" here means that an unqualified reference to a
 name attempts to find the name in the namespace.
 
 Although scopes are determined statically, they are used dynamically. At any
-time during execution, there are at least three nested scopes whose namespaces
-are directly accessible:
+time during execution, At any time during execution, there are various nested
```

Comment:
```suggestion
time during execution, At any time during execution, there are 3 or 4 nested
```
The list that follows has 4, 1 of which is only conditionally present, making 3 or 4.  I said more on the issue.